credentials/tls/certprovider/pemfile/watcher_test.go

```diff
@@ -65,6 +67,10 @@ func compareKeyMaterial(got, want *certprovider.KeyMaterial) error {
 		return fmt.Errorf("keyMaterial roots = %v, want %v", gotR, wantR)
 	}
 
+	if gotBundle, wantBundle := got.SPIFFEBundleMap, want.SPIFFEBundleMap; !reflect.DeepEqual(gotBundle, wantBundle) {
```

Comment:
Please don't use `reflect.DeepEqual`. Please use the `cmp` package instead. See: go/go-style/decisions#types-of-equality

Reply:
Fixed